spec/adapters/alma_adapter_spec.rb

```diff
@@ -260,6 +261,13 @@
       expect(item[:status_label]).to eq "On-site Access"
     end
 
+    it "reports on-site access for some specific (map) locations" do
+      FactoryBot.create(:map_location, code: 'lewis$mapmc', label: 'Lewis Library - Map Collection. Map Case')
+      availability = adapter.get_availability_one(id: "9968442613506421")
```

Comment:
@hectorcorrea we will have a meeting with the maps team to update the list of the locations. We should update the fixture after the meeting  as well, unless you want to use a lewis$map fixture which is the code they use for the map location for sure.

Reply:
Yeah, we can totally update this list at anytime.